pages/restake/lst-concepts.mdx

```diff
@@ -1,41 +1,53 @@
+- **Permissionless unbonding**: Under specific conditions, such as when a pool is blocked or destroying, users can unbond their tokens permissionlessly.
 
-Tangle's reward system deducts a fee from the underlying staking rewards to support the protocol. The remaining rewards are distributed proportionally to liquid stakers.
+### 4. **Withdrawals**
 
-### Liquidity Challenges
+- **Withdraw Unbonded**: Once tokens are unbonded, users can withdraw their funds using the `withdraw_unbonded` function. This function can also be used permissionlessly if certain conditions are met (e.g., the pool is destroying, or the caller is the depositor and the only member left).
 
-Operation-specific LSTs may face liquidity issues in the DeFi ecosystem. Tangle addresses this by encouraging integrations that support trading between LSTs from the same underlying protocol.
+### 5. **Pool Management**
 
-### Security and Slashing
+- **Create Pools**: Users can create new delegation pools with the `create` function. A pool requires an initial deposit, and the creator can set the roles for root, nominator, and bouncer. Pools can also be created with a previously used pool ID using the `create_with_pool_id` function.
+- **Pool State Management**: The state of a pool can be modified using the `set_state` function. Pools can have various states, such as open, blocked, or destroying. Once a pool enters the destroying state, it cannot change its state again.
 
-LSTs serve as collateral for outsourced security and slashing. This ensures network security while enabling innovative staking and restaking solutions.
+### 6. **Rewards and Commission**
 
-### Multi-Chain Support
+- **Commission**: Pools can set a commission percentage using the `set_commission` function. This determines the amount of rewards that are paid to the pool's commission account.
+- **Claiming Commission**: Pending commission can be claimed using the `claim_commission` function, which adds the claimed amount to the total claimed commission and resets pending commission to zero.
+- **Maximum Commission**: The maximum allowable commission can be set with the `set_commission_max` function, ensuring that future commissions are bound by this limit.
+- **Commission Change Rate**: The rate at which commission changes can be controlled using the `set_commission_change_rate` function.
 
-Tangle's liquid staking protocol supports multiple blockchains, including Ethereum and Polkadot, allowing users to stake assets across different networks.
+### 7. **Adjusting Pool Deposits**
 
-### Governance and Community Involvement
+- Pools must maintain an existential deposit (ED) in the reward account to ensure proper functionality. The `adjust_pool_deposit` function allows users to either top up the deficit or withdraw any excess deposit from the pool.
```

Comment:
When a pool is created do we deposit the ED into it? When the pool's fees are claimed do we ensure we leave enough to maintain its ED?